samconfig.toml

```diff
@@ -0,0 +1,11 @@
+version = 0.1
+[default]
+[default.deploy]
+[default.deploy.parameters]
+stack_name = "instance-scheduler"
```

Comment:
What is creating these assets and how are they managed?

Reply:
This file is not used at the moment. What it does is it allows for easy deployment if you wanted to deploy the lambda to an environment (say Sprinkler) using the `sam deploy` command. I think it could be useful if someone wanted to experiment with lambda deployment on a sandbox account, but let me know if you think it's better to remove it.